Add page for highlighted references

konradpa opened the pull request:
This adds a page to the "misc" section where we can highlight references we would like to suggest. (#95)

So far I added the offical GitHub documentation and the "Pro Git" book. What else do you think is missing?

Closes #151 
Closes #95

lnnrtwttkhn commented on _quarto.yml:
```diff
@@ -45,10 +45,12 @@ book:
     - chapters/rewriting-history.qmd
     - misc/exercises.qmd
     - misc/cheatsheet.qmd
+    - misc/highlighted_references.qmd
```
Please rename to `resources`. Use `git mv`

lnnrtwttkhn commented:
Thanks, I renamed the files a bit, see above.

> What else do you think is missing?

- [ ] https://happygitwithr.com/
- [ ] https://handbook.datalad.org/en/latest/, see #151
- [ ] https://book.the-turing-way.org/reproducible-research/vcs
- [ ] https://swcarpentry.github.io/git-novice/

After you added those, I would review the descriptions in more detail. Thanks!

konradpa commented:
> Thanks, I renamed the files a bit, see above.
> 
> > What else do you think is missing?
> 
> * [x]  https://happygitwithr.com/
> * [x]  https://handbook.datalad.org/en/latest/, see [add datalad handbook to resources? #151](https://github.com/lnnrtwttkhn/version-control-book/issues/151)
> * [x]  https://book.the-turing-way.org/reproducible-research/vcs
> * [x]  https://swcarpentry.github.io/git-novice/
> 
> After you added those, I would review the descriptions in more detail. Thanks!

Added those 👍

lnnrtwttkhn commented:
Very nice PR, thank you @konradpa!

Ready to merge! 🚀
